src/data/calendar.js

```diff
@@ -0,0 +1,136 @@
+/* global getBoxingDays, getFrenchBankHolidays */
+(function(exports) {
+  "use strict";
+
+  function guessTypeFromComment(comment) {
+    var KNOWN_TYPES = ['RTT', 'CP', 'CS'];
+    var words = {'sick': 'M', 'malad': 'M', 'patho': 'M',
+                 'wedding': 'CS', 'mariage': 'CS',
+                 'maternité': 'CS', 'parental': 'CS', 'paternité': 'CS',
+                 'sans solde': 'CS'};
```

Comment:
Actually I put it likes that to make it more readable :)
All the word that are related are on the same line.